src/user-guide/writing-suites/runtime.rst

```diff
@@ -376,8 +372,7 @@ run time to the home directory of task owner on the task host.
 How Tasks Get Access To The Suite Directory
 -------------------------------------------
 
-Tasks can use ``$CYLC_SUITE_DEF_PATH`` to access suite files on
-the task host, and the suite bin directory is automatically added
+The suite bin directory is automatically added
```

Comment:
```suggestion
The workflow bin directory is automatically added
```
Although I guess the mass workflow/suite change has not been done so this is unimportant I guess!

Reply:
I'm going to have to keep correcting myself for years...